packages/components/README.md

```diff
@@ -1,104 +1,7 @@
-# Experience Builder Components
+# @contentful/experiences-components-react
 
-> Experience Builder is currently in a private alpha and not available publicly. If you are interested in participating in the alpha, please reach out to your Contentful account team.
+This package provides the default structure and basic components for the Experiences SDK. You can use these components directly to start building your experiences, or as examples to create your own components.
```

Comment:
Let's also link somewhere for "Experiences SDK" considering a customer ends up here and wants to learn more? Maybe

```suggestion
This package provides the default structure and basic components for the [Experiences SDK](https://www.contentful.com/developers/docs/experiences/set-up-experiences-sdk/). You can use these components directly to start building your experiences, or as examples to create your own components.
```

or a link to the actual GH readme.

Reply:
Yes, that's a good idea. Updated